svdbs.md

```diff
@@ -1,9 +1,13 @@
 # Overview
+Location: NEFSCDB1
 
+Schema: SVDBS 
 SVDBS contains information from the fishery independent surveys.
 
 Most of the information there has been processed to some degree.
 
+For access, email paul.kostovick@noaa.gov
```

Comment:
Recommend linking the whole app schema inventory somewhere: https://docs.google.com/spreadsheets/d/15FtGnNUgct7mTsRpPP9xX4BLkceY7SfMnZvdA_kjlxY/edit#gid=1754518543

Reply:
It's in the main readme.

Reply:
Does anyone in the branch use svdbs?

Reply:
> Does anyone in the branch use svdbs?

I use it to pull data for age-length keys for the Groundfish Bioeconomic Rec model. I'm guessing @acarrharris does as well for Scup, SF, and BSB.

Reply:
they are.